### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by axiom**
I will push comments here... (not intended for merging)

### CODE REVIEW

File: elasticsearch/slurp.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ type Documenter interface {
 	Document() (map[string]interface{}, error)
 }
 
-type Operationer interface {
+type Operationer interface { // Actioner?
```
Comment (balboah) [inline elasticsearch/slurp.go:27]: Yeah started like that, kind of weird name though?

File: mongodb/timestamp.go [hunk 2]
```diff
@@ -12,7 +12,7 @@ import (
 type Timestamp bson.MongoTimestamp
 
 // Time converts a mongo timestamp to Time with UTC selected as timezone.
-func (t *Timestamp) Time() *time.Time {
+func (t *Timestamp) Time() *time.Time { // (majoh): Why pointers?
```
Comment (balboah) [inline mongodb/timestamp.go:15]: That doesn't make sense for Time no, it's on the list of possible tweaks

File: mongodb/operation.go [hunk 3]
```diff
@@ -173,4 +173,10 @@ type Manipulator interface {
 	Manipulate(doc *map[string]interface{}) error
 }
 
+type ManipulatorFunc func(*map[string]interface{}) error
```
Comment (balboah) [inline mongodb/operation.go:184]: Pushed this :)

File: main.go [hunk 4]
```diff
@@ -18,19 +18,15 @@ import (
 )
 
 var (
-	mongoServer  = flag.String("mongo", "localhost", "Specific server to tail")
-	mongoInitial = flag.Bool(
-		"initial",
-		false,
-		"True if we want to do initial sync from the full collection, otherwise resume reading oplog")
+	// I think I prefer long lines here, it's more readable than the "random" wrap.
```
Comment (balboah) [inline main.go]: Done

File: main.go [hunk 5]
```diff
@@ -81,7 +77,7 @@ func main() {
 		for op := range mongoc {
 			select {
 			// Wrap all mongo operations to comply with ES interface, then send them off to the slurper.
-			case esc <- &mongodb.EsOperation{
+			case esc <- &mongodb.EsOperation{ // This is a bit awkward.... maybe create it outside of the select?
```
Comment (balboah) [inline main.go]: done

**Comment by balboah** [comment]
About time it was closed!

**Comment by kristinn** [comment]
Hahaha! :heart:

**Comment by kristinn** [comment]
LOLUR!

**Pull request closed without merging**